docs/web/index.md

```diff
@@ -406,13 +457,25 @@ So far this covers:
 All these types can be read and written in JSON as ISO 8601 formats.
 Reading `java.time.ZonedDateTime` is configured to be tolerant so that added nanoseconds or missing
 milliseconds or missing seconds are supported.
+
 `@com.fasterxml.jackson.annotation.JsonFormat(pattern = "...")` should not be used for customizing
 serialization because it breaks tolerant reading of formatting variants. If a specific field should
 be serialized with milliseconds, it must be annotated with
 `@com.fasterxml.jackson.databind.annotation.JsonSerialize(using = Iso8601Serializer.WithMillis.class)`
 . If a specific field should be serialized with nanoseconds, it must be annotated with
 `@com.fasterxml.jackson.databind.annotation.JsonSerialize(using = Iso8601Serializer.WithNanos.class)`
 
+**Differences to the known [SDA Dropwizard Commons configuration](https://github.com/SDA-SE/sda-dropwizard-commons/tree/master/sda-commons-server-jackson)**
+- `java.time.ZonedDateTime` fields are serialized with seconds by default.
+  There is no other global configuration for **java.time.ZonedDateTime** serialization available.
+- **Less modules are activated for foreign frameworks**. Compared to SDA Dropwizard Commons,
+  **GuavaExtrasModule, JodaModule, AfterburnerModule and CaffeineModule** are not registered anymore. 
```

Comment:
Compare to https://github.com/SDA-SE/sda-dropwizard-commons/blob/master/sda-commons-server-jackson/src/main/java/org/sdase/commons/server/jackson/ObjectMapperConfigurationUtil.java#L112

The `AfterburnerModule` is not registered anymore. Please remove it from the list.

Reply:
Done 👍🏼